src/test/java/io/fabric8/maven/docker/service/RunServiceTest.java

```diff
@@ -277,6 +279,38 @@ public void testWithException() throws Exception {
         runService.stopContainer(container, createImageConfig(SHUTDOWN_WAIT, 0), false, false);
     }
 
+    @Test
+    public void testWithMultipleStopExceptions() throws Exception {
+        GavLabel testLabel = new GavLabel("Im:A:Test");
+
+        String firstName = "first-container:latest";
+        ImageConfiguration first = new ImageConfiguration();
+        first.setName(firstName);
+
+        String secondName = "second-container:latest";
+        ImageConfiguration second = new ImageConfiguration();
+        second.setName(secondName);
+
+        tracker.registerContainer(firstName, first, testLabel);
+        tracker.registerContainer(secondName, second, testLabel);
+
+        LogOutputSpecFactory logOutputSpecFactory = new LogOutputSpecFactory(true, true, null);
+
+        new Expectations(){{
+           docker.stopContainer(firstName, 0); result = new DockerAccessException("TEST one");
+           docker.stopContainer(secondName, 0); result = new DockerAccessException("TEST two");
+        }};
+
+        runService = new RunService(docker, queryService, tracker, logOutputSpecFactory, log);
+
+        try {
+            runService.stopStartedContainers(false, true, true, testLabel);
+            fail("Should have thrown exception");
+        } catch (DockerAccessException | ExecException e) {
+            assertEquals(e.getLocalizedMessage(), "(TEST two,TEST one)");
```

Comment:
nit, these arguments should be in opposite order: `assertEquals(expected, actual)`